pkgs/applications/editors/emacs/sources.nix

```diff
@@ -31,8 +32,8 @@ let
             }
           );
           "macport" = (
-            fetchFromBitbucket {
-              owner = "mituharu";
```

Comment:
Was this abandoned, or it is just dormant?

Reply:
To my knowledge, it is only dormant. There doesn't appear to be a branch for Emacs 30, so I swapped sources to a fork until something stable comes out of the woodwork.